cmd/check_container_test.go

```diff
@@ -574,4 +137,26 @@ certification_project_id: mycertid`
 			})
 		})
 	})
+
+	Context("When instantiating a checkContainerRunner", func() {
+		var cfg *runtime.Config
+
```

Comment:
The `cfg` in this block is missing initialization now, which causes tests to panic. I believe the block that is missing looks like this:

```
		BeforeEach(func() {
			cfg = &runtime.Config{
				Image:          "quay.io/example/foo:latest",
				ResponseFormat: formatters.DefaultFormat,
			}
		})
```

Reply:
Fixed.